eng/pipelines/templates/variables/vmr-stage.yml

```diff
@@ -0,0 +1,49 @@
+variables:
+- ${{ if ne(parameters.vmrBranch, '') }}:
```

Comment:
Don't we need to specify the template parameters?

Reply:
thanks yeah I think you're right, will fix.